pkg/kubectl/resource/builder.go

```diff
@@ -709,7 +709,7 @@ func (b *Builder) visitorResult() *Result {
 	}
 
 	if len(b.resources) != 0 {
-		return &Result{err: fmt.Errorf("resource(s) were provided, but no name, label selector, or --all flag specified")}
+		return &Result{err: fmt.Errorf("resource(s) were provided, but no name, label selector, field selector or --all flag specified")}
```

Comment:
not all components that use the resource build provide a field selector option... is this message going to be confusing?